corporate/tests/test_stripe.py

```diff
@@ -2373,10 +2373,22 @@ def test_update_billing_method_of_current_plan(self) -> None:
         update_billing_method_of_current_plan(realm, True)
         plan.refresh_from_db()
         self.assertEqual(plan.charge_automatically, True)
+        realm_audit_log = RealmAuditLog.objects.filter(
+            event_type=RealmAuditLog.REALM_BILLING_METHOD_CHANGED
+        ).order_by("-id")[0]
```

Comment:
This can be `last()` like I mentioned in previous commit review. Also update other occurrences as well.